maplibre-adapter/src/main/java/com/amplifyframework/geo/maplibre/AmplifyMapLibreAdapter.kt

```diff
@@ -78,8 +66,6 @@ object AmplifyMapLibreAdapter {
      * @param style Amplify map style to use
      * @param callback Callback to trigger upon successfully loading map style
      */
-    @JvmStatic
-    @JvmOverloads
```

Comment:
i think we should keep this no? `setStyle(map, callback)` feels like a pretty desirable API for Java devs so that they don't have to worry about passing in map style

Reply:
Discussed offline that adapter will be internal